### ISSUE TITLE
The mod Fucks up the maps.

### ISSUE THREAD

**Issue by SpanishBnn** [Sheights/StarboundSimpleVoreMod#112]
So.
Ive discovered quite big issues with the mod and leaving the planet.
There are no issues WHILE youre in the planet, but once you leave it, you cannot come back.Literally, you cant; because you will instantly be sended to the spaceship once the game starts loading the part where the vore mod has been affected.
This are a list of what makes it blow up so badly 
-More than a certain ammount of tenants
-Disconnecting while getting vored
-TPeing yourself to spaceship while getting vored
The vore "items" (those NPCs you can buy but remain static)  gives no problems, but tenants are pretty much a plague of glitches.I tried several times to make a world with a house where every each of the NPCs are in for my own amusement and once I go out the planet and back, the game freezes on the loading screen or it just sends you back to the spaceship
Maybe im the only one.This errors are too big to let it slide.The thing is= Ive lost already 3 worlds because of those reasons, and with that I wasted all my time input tenants which, honestly, is BS

**Comment by Rixunie** [comment]
I've been unable to reproduce this. I've tested it with Frackin' and space stations as well. Perhaps you have a mod conflict or another mod altering tenants?

**Comment by SpanishBnn** [comment]
the only mod that I have that maybe alter tenants is Frackin' Universe.The rest of them are HUDs, or just new weapons/races for players use.
I had this problem just as I reinstalled the game

**Comment by KH007** [comment]
That would probably be why, then. Frackin' messes directly with the files, rather than just acting like any other mod. Did you reinstall Frackin' in the reinstalled game?

**Comment by SpanishBnn** [comment]
I did, mainly because Steam does it directly.
Any way to have a save with Frackin and another without?

**Comment by KH007** [comment]
Yes; I did it with a modset that doesn't include Frackin. You go into the files of the fresh game without frackin and duplicate the files before moving the copies into a separate folder elsewhere. When you want to revert to the pre-frackin version, just swap the frackin-version files with the pre-frackin files.

**Comment by SpanishBnn** [comment]
Well, another issue just show up.After uninstalling Frackin universe, ive reinstalled the game, installed the mod, and now when I try to get into the Foodhall it doesnt let me.Its sends me back to the spaceship

**Comment by KH007** [comment]
The only solution to that is to remove the world instance file and re-load the foodhall, I think.

**Comment by SpanishBnn** [comment]
Allright everything fixed.Though now I'm scared to do anything in the game that could blow up my game lol

**Comment by Noinfo999** [comment]
I know this is old but it happened to me when i left the planet but i forgot to release the prey. Maybe you just forgot it was in there.
